PurchasesCoreSwift/LocalReceiptParsing/DataConverters/ArraySlice_UInt8+Extensions.swift

```diff
@@ -32,8 +32,8 @@ extension ArraySlice where Element == UInt8 {
         return String(bytes: self, encoding: .utf8)
     }
 
-    func toDate(dateFormatter: ISO3601DateFormatter) -> Date? {
-        return dateFormatter.date(fromBytes: self)
+    func toDate(dateFormatter: DateFormatter) -> Date? {
+        return dateFormatter.iso8601SecondsOrMillisecondsDate(fromBytes: self)
```

Comment:
not 100% sure it's needed to default to milliseconds, but deciding on one or the other would require a decent amount of testing.